app/views/devise/registrations/edit.html.erb

```diff
@@ -38,6 +38,21 @@
 
 <h3>Cancel my account</h3>
 
-<p>Unhappy? <%= button_to "Cancel my account", registration_path(resource_name), data: { confirm: "Are you sure?" }, method: :delete %></p>
+<% if resource.class.require_password_to_destroy %>
+  <%= form_for(resource, as: resource_name, url: registration_path(resource_name), html: { method: :delete }) do |f| %>
```

Comment:
This form is exactly the same as the previous one. This will generate invalid HTML - multiple elements with the same ID.
<img width="559" alt="screen shot 2018-12-04 at 18 41 36" src="https://user-images.githubusercontent.com/3727827/49471637-96453900-f7f4-11e8-93fa-76c8d630b2ee.png">

One solution that I see is to rename this form so that it won't cause conflicts with the previous one.